### PR TITLE
 Fix the version of linebot/version.go and  add a shell script to prevent to forget update of linebot/version.go .

### PR DESCRIPTION
**Pull request by suzuki-shunsuke**

### CODE REVIEW

File: script/tag.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  echo "TAG must start with 'v'"
+  exit 1
+fi
+
+echo "cd `dirname $0`/.."
+cd `dirname $0`/..
+
+echo "create linebot/version.go"
+cat << EOS > linebot/version.go
+// Copyright 2016 LINE Corporation
```
Comment (sugyan) [inline script/tag.sh]: 2018?

Comment (suzuki-shunsuke) [reply script/tag.sh]: Oh, sorry..

Comment (suzuki-shunsuke) [reply script/tag.sh]: Oh, I misunderstood.
This 2018 is correct. linebot/version.go has created before 2018.
I should add LICENSE at the head of tag.sh.

Comment (suzuki-shunsuke) [reply script/tag.sh]: I have added the license.
https://github.com/line/line-bot-sdk-go/pull/103/commits/87cba7bae8f23c6f8a887b2dbfd1f2f2b8878cc0

Comment (sugyan) [reply script/tag.sh]: ah, I was wrong, thank you 🙇

**Pull request merged**